### ISSUE TITLE
Bug - 419 error in manager portal

### ISSUE THREAD

**Issue by vid** [GCTC-NTGC/TalentCloud#834]
# Description
Sometimes, after submitting the login form on the manager portal, a 419 error is returned.

# Steps to Reproduce
Steps to reproduce the behavior. 
1. Go to the manager portal (as Lauren)
2. Submit login form
3. 419 error

This happens several times in a row.

**Comment by xander-miller** [comment]
Screen shot of error with exception occurs when restarting a stale session. 

![419 Error Exception](https://user-images.githubusercontent.com/663395/55077169-7371b580-506d-11e9-9ad2-e7f095136bc5.PNG)

``` /var/www/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php#82
Illuminate\Session\TokenMismatchException
            });
        }

        throw new TokenMismatchException;
    }

    /** 
```

**Comment by tristan-orourke** [comment]
The idea came up in StandUp today that re-opening a form in a new window might also lead to this bug.

**Comment by gggrant** [comment]
Appears to be resolved by #945
